src/Roslyn.Suppressions/Roslyn.Suppressions/InProcess/RoslynSettingsFileSynchronizer.cs

```diff
@@ -98,7 +97,7 @@ private void SuppressedIssuesMonitor_SuppressionsUpdateRequested(object sender,
             catch (Exception ex) when (!ErrorHandler.IsCriticalException(ex))
             {
                 // Squash non-critical exceptions
-                logger.LogDebugExtended(ex.ToString());
```

Comment:
This was the only place that used LogDebugExtended, which was tricky to implement in the Logger:
ILogger.LogDebug receives params, and you cannot put optional args [CallerFilePath] and [CallerMemberName] after them. Since this was the only usage, I left it out for now.

Reply:
We only really used it for internal builds where we needed extra info.